plugins/teststeps/teststeps_test.go

```diff
@@ -0,0 +1,369 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
I see these tests takes long:
```
xaionaro@void:~/go/src/github.com/facebookincubator/contest$ go test ./plugins/teststeps/ -count=1
ok      github.com/facebookincubator/contest/plugins/teststeps  4.343s
```

May be we should add a build tag like `longtests` or something like that. @marcoguerri : what do you think?